.travis.yml

```diff
@@ -1,51 +1,42 @@
 # Travis can building for Linux and macOS
-matrix:
+language: python
+python:
+  - 3.5
+  - 3.6
+  - 3.7
+addons:
+  apt:
+    update: true
+    packages:  # Linux packages needed for Qt to work.
+    - libxkbcommon-x11-0
+services: xvfb
```

Comment:
Nice! I didn't realised this was option, this will help keep the rest of the script more generic between Linux and macOS.